Multisig/UI/Settings/AddressBook/AddressBookListTableViewController.swift

```diff
@@ -106,6 +106,11 @@ class AddressBookListTableViewController: LoadableViewController, UITableViewDel
         let cancelButton = UIAlertAction(title: "Cancel", style: .cancel, handler: nil)
         alertController.addAction(cancelButton)
 
+        if let popoverPresentationController = alertController.popoverPresentationController {
+            //TODO: What view should be used? tableView is wrong
+            popoverPresentationController.sourceView = tableView
```

Comment:
you can try using `..Controller.barButtonItem = menuButton`

Reply:
![image](https://user-images.githubusercontent.com/246473/159052451-d00504a1-6098-435f-8257-5d4c9dcb8331.png)

That is exactly, what I was looking for!